ArcGIS-toolbox/ASF_Tools.pyt

```diff
@@ -861,7 +861,7 @@ class RGBDecomp(object):
         arcpy.AddMessage("Pixel cleanup complete. Generating spatial masks...")
 
         # Generate spatial masks based on red/blue threshold
-        rb_thresh = math.pow(10, rb_thresh_db / 10)
+        rb_thresh = math.pow(10, float(rb_thresh_db) / 10)
```

Comment:
It would be more robust to change the parameter format to GPDouble from GPLong (line 670). I would also change the displayName to read "...(default: -24.0 dB)" 
Also change line 760 to be -24.0 instead of -24, for clarity.
This solves the integer division problem, and also allows users finer control of the threshold parameter by supporting decimals.